lib/firefox/webdriver/firefoxPreferences.js

```diff
@@ -12,35 +12,9 @@ module.exports = {
   // Use hidden TTCP
   'dom.performance.time_to_contentful_paint.enabled': true,
```

Comment:
What about when we do this release switch to use your defaults instead and release it as a major release? Right now these are` --firefox.mozillaProPreferences` but I think we can just make those default in later PR or do you see a problem with that?

Reply:
I'll undo these changes and we can do that in a later PR.